### PR TITLE
[FS-154088]: node 16 requisite changes in freshservice

### PR DESCRIPTION
**Pull request by SivaSathyamurthy**
node 16 requisite changes in freshservice

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,7 +46,8 @@
     "ember-resolver": "^4.5.5",
     "ember-source": "2.18.2",
     "eslint-plugin-ember-suave": "^1.0.0",
-    "loader.js": "^4.6.0"
+    "loader.js": "^4.6.0",
+    "sass": "^1.77.6"
```
Comment (preethi-sathiyanarayanan) [inline package.json:50]: Add details and screenshots explaining the need for this change

**Review by preethi-sathiyanarayanan — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (SivaSathyamurthy) [reply package.json:50]: https://www.npmjs.com/package/ember-cli-sass/v/8.0.1

**Review by vigneshk-fd — APPROVED** [review]

**Review by sreeram-cv — APPROVED** [review]

**Comment by sreeram-cv** [comment]
please add reason why no impact here and in itil PR
- dev dependencies are only for dummy app, no impact in product + ember documentation screenshot
- ensure using npm as well + npm ls screenshot

**Comment by SivaSathyamurthy** [comment]
1) Reason behind no impact is, Since it is added as **dev dependency, it has no impact on product**. 

**Reference screenshot**
image

**Ref doc**
https://cli.emberjs.com/release/writing-addons/addon-dependencies/
 
 
 




**2) npm ls screenshot comparision:**

**- v16.20.2**
image

**- v12.19.0**
image

**Pull request merged**